apps/gateway/test/champions_test.exs

```diff
@@ -477,14 +477,17 @@ defmodule Gateway.Test.Champions do
                rate.currency_id == gems_currency_id && rate.rate > 0
              end)
 
-      # Wait for the afk rewards to increase
-      milliseconds_to_wait = 2000
-      :timer.sleep(milliseconds_to_wait)
-
       # Claim afk rewards
       gold_before_claiming = Currencies.get_amount_of_currency_by_name(advanced_user.id, "Gold")
       gems_before_claiming = Currencies.get_amount_of_currency_by_name(advanced_user.id, "Gems")
 
+      # Simulate waiting 2 seconds before claiming the rewards
+      seconds_to_wait = 2
+      {:ok, advanced_user_db} = Users.get_user(advanced_user.id)
+      advanced_user_db
+      |> GameBackend.Users.User.changeset(%{last_afk_reward_claim: DateTime.utc_now() |> DateTime.add((-1) * seconds_to_wait, :second)})
```

Comment:
You won't be able to commit this since after formatting it updated it to `(-1 * seconds_to_wait)`. I still think removing the 1 altogether makes the code clearer though